ts795/pack-your-bags
# Add client side javascript to do a post to /login

Testing
1. Type in values in username and password fields
2. Click Sign in
![image](https://user-images.githubusercontent.com/29023131/128121435-b2c52e9e-9928-4b4e-bc87-2cf616a6d648.png)
3. Check that /trip is shown
![image](https://user-images.githubusercontent.com/29023131/128128016-9e0db84e-3d7e-49d5-8216-efa2ee73b5ed.png)
